model/proto/model.proto

```diff
@@ -158,3 +158,13 @@ message Batch {
       (gogoproto.nullable) = true
     ];
 }
+
+message DependencyLink {
+  string parent = 1;
+  string child = 2;
+  uint64 call_count = 3;
+  string source = 4 [
+    (gogoproto.customtype) = "DependencyLinkSource",
```

Comment:
I thought that this would be the best of both worlds re: the conversation above.

Reply:
well, if we do this, I would recommend adding a simple marshaling/unmarshaling test, similar to `model/ids_test.go`

Reply:
Actually, not just for this, but also for importing `model.Span`. I found it not particularly useful just adding gogoproto annotations without testing that serialization works as expected

Reply:
I'm not following what you mean by "importing `model.Span`". In the plugin proto file? I don't believe that we actually use gogoproto annotations for importing Span there?

Reply:
I mean this:
```
message GetTraceRequest {
    bytes trace_id = 1 [
      (gogoproto.nullable) = false,
      (gogoproto.customtype) = "github.com/jaegertracing/jaeger/model.TraceID",
      (gogoproto.customname) = "TraceID"
    ];
}
```
From my previous experience  with gogoproto, even though `protoc` may compile it, you won't know if it will serialize correctly until you try it in unit tests.

Reply:
and similar for spans